pkg/lib/serverTime.js

```diff
@@ -469,7 +469,7 @@ export function ServerTimeConfig() {
                 body = <div>{cockpit.format(_("Synchronized with $0"), ntp.server)}</div>;
         } else {
             if (ntp.server) {
-                icon = <Spinner size="md" isSVG />;
+                icon = <Spinner size="md" />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230425-151143-725ccf60-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)